keras_autodoc/get_signatures.py

```diff
@@ -21,7 +20,15 @@ def get_signature_start(function):
 
 
 def get_signature_end(function, class_aliases={}):
-    signature_end = Signature(function).format_args()
+    signature = inspect.signature(function)
+    signature = signature.replace(
+        parameters=[
+            param.replace(annotation=inspect.Signature.empty)
+            for param in signature.parameters.values()
+        ],
+        return_annotation=inspect.Signature.empty
+    )
+    signature_end = str(signature)
     if utils.ismethod(function):
         signature_end = signature_end.replace('(self, ', '(')
         signature_end = signature_end.replace('(self)', '()')
```

Comment:
Just below this line, there is the substitution for aliases (for example, a type hint will display `keras.engine.base.Layer` but this will replace it with `keras.layers.Layer`). We should move it now to the place were the docstring is generated.

Reply:
Class aliases seem to be used in a lot of places in that file; I'm not sure what refactor you have in mind here exactly, sorry.

Reply:
Sure thing, I'll do it after this PR is merged.